d2/src/main/java/com/linkedin/d2/xds/XdsClient.java

```diff
@@ -26,7 +26,7 @@ public abstract class XdsClient
 {
   private static final String D2_NODE_TYPE_URL = "type.googleapis.com/indis.D2Node";
```

Comment:
do we need to change NODE_TYPE to URI_TYPE?

Reply:
No the D2Node type was still used for clusters and services